packages/ia-topnav/www/index.html

```diff
@@ -1,5 +1,15 @@
 <link rel="stylesheet" href="https://archive.org/includes/build/css/archive.min.css">
 
+<script type="importmap">
+  {
+    "imports": {
+      "@internetarchive/ia-wayback-searchx": "https://esm.archive.org/@internetarchive/ia-wayback-search",
```

Comment:
QUESTION: What is this `ia-wayback-searchx`? Seems like an effective commenting out? If so, can we just remove this line?

Reply:
yes, sorry, an oversight indeed!  i removed it.  (artifact while testing out import maps)